jsk_fetch_robot/jsk_fetch_startup/launch/fetch_teleop.xml

```diff
@@ -2,28 +2,41 @@
   <!-- mainly copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
   <arg name="enable_auto_dock" default="true" />
   <arg name="joy_device" default="/dev/ps3joy"/>
-  <arg name="disable_arm_teleop" default="true"/>
 
+  <!-- copied from $(find fetch_bringup)/launch/include/teleop.launch.xml -->
```

Comment:
need to launch `joy_node` because of `launch_teleop:=false` is set at `/etc/ros/indigo/robot.launch`, see https://github.com/fetchrobotics/fetch_robots/pull/42

Reply:
I added the comment.